### PR TITLE
Check if user is allowed to see detail view link in agenda list.

### LINKED ISSUE #1886: 403 after try to open customslide detail view for normal users
Should be possible for normal users which has agenda.can_see permissions! Please fix for 2.0b2.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
Fixes #1886

### CODE REVIEW

File: openslides/agenda/static/js/agenda/site.js [hunk 1]
```diff
@@ -85,6 +86,20 @@ angular.module('OpenSlidesApp.agenda.site', ['OpenSlidesApp.agenda'])
...
+                case 'core/customslide':
+                    return operator.hasPerms('core.can_manage_projector');
+                case 'motions/motion':
+                    return operator.hasPerms('motions.can_see');
+                case 'assignments/assignment':
+                    return operator.hasPerms('assignments.can_see');
+                default:
+                    return false;
+            }
+        }
```
Comment (normanjaeckel) [inline openslides/agenda/static/js/agenda/site.js]: Add semicolon.

File: openslides/agenda/models.py [hunk 2]
```diff
@@ -233,7 +233,7 @@ class Item(RESTModelMixin, models.Model):
     The parent item in the agenda tree.
     """
 
-    weight = models.IntegerField(default=0)
+    weight = models.IntegerField(default=10000)
```
Comment (normanjaeckel) [inline openslides/agenda/models.py]: Missing migrations file.

Comment (emanuelschuetze) [inline openslides/agenda/models.py]: added

**Comment by emanuelschuetze** [comment]
All done.

**Comment by normanjaeckel** [comment]
Travis failed

**Comment by emanuelschuetze** [comment]
Fixed travis.

**Comment by emanuelschuetze** [comment]
Fixed travis.

**Comment by emanuelschuetze** [comment]
Fixed travis.

**Pull request merged**